watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/amo_1595_photothermal_membrane_candoP/amo_1595_case_study.yaml

```diff
@@ -8,7 +8,7 @@ defined_flows:
     value: 0.5
     units: USD_2020/kg
   water:
-    value: 1.0
+    value: 1.0e-8
```

Comment:
Why changes this value? It seems very little cost of water

Reply:
I discussed this with @adam-a-a - I want the default value to be 0, but if you put the value as exactly 0 or 0.0, there is a bug in pyomo which causes an error. So instead I set it to something infinitesimally small.

Reply:
It probably doesn't matter -- though if this is to be "zero" I would choose a smaller value (`1e-10` or lower)

Reply:
If the water cost is 0, then why not directly delete this registered flow? Does it use in any place other than operating cost?

Reply:
@andrewlee94 - done

@bknueven - done on latest commit

@luohezhiming - The intent is to keep the parameter available in case we want to run a sensitivity analysis in which the sale price of water is > 0. We just want the default value to be zero.